macro/src/lib.rs

```diff
@@ -117,9 +117,15 @@ pub fn http_component(_attr: TokenStream, item: TokenStream) -> TokenStream {
             async fn handle_response<R: ::spin_sdk::http::IntoResponse>(response_out: ::spin_sdk::http::ResponseOutparam, resp: R) {
                 let mut response = ::spin_sdk::http::IntoResponse::into_response(resp);
                 let body = std::mem::take(response.body_mut());
```

Comment:
I know you didn't change this, but:

```suggestion
                let body = ::std::mem::take(response.body_mut());
```